TvMaterialCatalog/gradle/libs.versions.toml

```diff
@@ -1,26 +1,30 @@
 [versions]
-agp = "8.3.1"
+agp = "8.3.2"
 coil-compose = "2.5.0"
+foundation = "1.7.0-beta07"
 kotlin = "1.9.0"
-core-ktx = "1.12.0"
-appcompat = "1.6.1"
-compose-bom = "2024.03.00"
+core-ktx = "1.13.1"
+appcompat = "1.7.0"
+compose-bom = "2024.06.00"
```

Comment:
We could update it to 2024.09.01
c.f. https://developer.android.com/develop/ui/compose/bom/bom-mapping